tests/SnowflakeTest.php

```diff
@@ -522,7 +522,7 @@ public function testGetTablesWithoutSchema(): void
         // add a table to a different schema (should not be fetched)
```

Comment:
it is fetched though - it's even in the expected data...

Reply:
sorry, the comment is incorrect.  schema is not set in this test so all tables from all schemas should be fetched.  The test name `testGetTablesWithoutSchema` is correct.